specification/security/resource-manager/Microsoft.Security/stable/2019-08-01/iotSecuritySolutions.json

```diff
@@ -455,6 +455,29 @@
         },
         "recommendationsConfiguration": {
           "$ref": "#/definitions/RecommendationConfigurationList"
+        },
+        "unmaskedIpLoggingStatus" : {
```

Comment:
How do you treat this property if it is set by someone and then not included in the next PUT request by someone with an older api-version or a previously published SDK for this api-version that doesn't know about the property?  

Given the default value this should revert to disabled if it is not specified which would be an unexpected effect for someone using an older api-version.  This would qualify as a breaking change and should be in a new api-version if so:

https://microsoft.sharepoint.com/:w:/r/teams/azureresourcemanagerteam/_layouts/15/guestaccess.aspx?e=xEcppo&share=EWXsAQ1yx25KkyYCeeWGUgwBSBxEdUDEbHi6FZ__U8EOQw

Reply:
Our SDK and REST docs weren't published yet (planned to be in the near future).
So customers aren't supposed to make calls to this API yet (they wouldn't have the tools to do so ... unless someone reversed engineered it 🤔).

Is it enough to avoid the api-version change?